Replicator/Worker.hh

```diff
@@ -228,7 +228,18 @@ namespace litecore { namespace repl {
 
 #pragma mark - INSTANCE DATA:
     protected:
-        void setMsgCollection(blip::MessageBuilder& msg, CollectionIndex);
+        static CollectionIndex getCollectionIndex(const blip::MessageIn&  msgIn) {
+            long l = msgIn.intProperty(kCollectionProperty, kNotCollectionIndex);
+            Assert(l <= kNotCollectionIndex);
```

Comment:
If you are going to do this, it would be worth checking that l is also >= 0

Reply:
Oh, great! We thought the same thing. See my earlier reply.